### PR TITLE
Bump JVM and Exposed versions to latest

### PR DESCRIPTION
**Pull request by williamboxhall**

### CODE REVIEW

File: src/main/kotlin/com/cultureamp/eventsourcing/RelationalDatabaseEventStore.kt [hunk 1]
```diff
@@ -126,7 +126,7 @@ class RelationalDatabaseEventStore<M : EventMetadata> @PublishedApi internal con
                             SequencedEvent(event, -1)
                         } else {
                             val insertedSequence = insertResult[eventsSinkTable.sequence]
-                            eventsSequenceStats.replace {
+                            eventsSequenceStats.upsert {
```
Comment (davidwheeler123) [inline src/main/kotlin/com/cultureamp/eventsourcing/RelationalDatabaseEventStore.kt:129]: is this the only `replace` in kestrel codebase?

File: src/main/kotlin/com/cultureamp/eventsourcing/jsonb.kt [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
 
     override fun setParameter(stmt: PreparedStatementApi, index: Int, value: Any?) {
-        stmt[index] = value as String
+        if (value is String) {
+            val pgObject = PGobject()
+            pgObject.type = "jsonb"
+            pgObject.value = value
+            stmt[index] = pgObject
+        } else {
+            stmt[index] = value!!
```
Comment (davidwheeler123) [inline src/main/kotlin/com/cultureamp/eventsourcing/jsonb.kt:22]: does stmt only accept not nullables? I wonder if there's another way to set null eg `stmt.setNull(index)`

**Review by davidwheeler123 — COMMENTED** [review]
Happy to approve if you have confirmed there's no other `replace()`s

File: src/main/kotlin/com/cultureamp/eventsourcing/RelationalDatabaseEventStore.kt (hunk 1 above)
Comment (williamboxhall) [reply src/main/kotlin/com/cultureamp/eventsourcing/RelationalDatabaseEventStore.kt:129]: yeah it's the only one

**Review by davidwheeler123 — APPROVED** [review]

**Pull request merged**